components/script/dom/element.rs

```diff
@@ -2575,6 +2602,21 @@ impl Element {
         self.set_click_in_progress(false);
     }
 
+    // https://html.spec.whatwg.org/multipage/#language
+    pub fn get_lang(&self) -> Atom {
+        if let Some(attr) = self.get_attribute(&ns!(xml), &local_name!("lang")) {
```

Comment:
This is super duper unpossible. I used many methods to try and make this generic but to no avail. The core problem here is the `LayoutJS<T>` pointers that needs to constantly be interchanged between `JS<T>`.